mne_connectivity/spectral/time.py

```diff
@@ -43,16 +46,31 @@ def spectral_connectivity_time(data, names=None, method='coh', indices=None,
             * 'sxy' : Cross-spectrum
 
         By default, the coherence is used.
+    average : bool
+        Average connectivity scores over Epochs. If True, output will be
+        an instance of ``SpectralConnectivity`` , otherwise
+        ``EpochSpectralConnectivity``.
     indices : tuple of array | None
         Two arrays with indices of connections for which to compute
         connectivity. I.e. it is a ``(n_pairs, 2)`` array essentially.
         If None, all connections are computed.
     sfreq : float
         The sampling frequency.
-    foi : array_like | None
-        Extract frequencies of interest. This parameters should be an array of
-        shapes (n_foi, 2) defining where each band of interest start and
-        finish.
+    fmin : float | tuple of float
+        The lower frequency of interest. Multiple bands are defined using
+        a tuple, e.g., (8., 20.) for two bands with 8Hz and 20Hz lower freq.
+        If None the frequency corresponding to an epoch length of 5 cycles
+        is used.
+    fmax : float | tuple of float
```

Comment:
I find it a little overwhelming myself as well. I tried to follow the API from `spectral_connectivity_epochs` as much as possible to make the functions feel more similar and thus easier to use. Indeed, some parameters do rule each other out and are only used in either `cwt_morlet` or `multitaper` mode.

I believe it would also be necessary to include documentation and perhaps an example on the different time-frequency decomposition methods, so that users (myself included) would understand the difference (advantages and disadvantages) between multitaper and complex Morlet wavelets. I would also consider adding filter-Hilbert as a time-frequency decomposition method, as some users might prefer it over multitaper and Morlet wavelets.